File: test/fixtures/crossmodule-ts-repo/agent.ts
import OpenAI from "openai";
import { SYSTEM_PROMPT } from "./prompts";

const client = new OpenAI();

const resp = client.chat.completions.create({
  model: "gpt-4o",
  messages: [
    { role: "system", content: SYSTEM_PROMPT },
    { role: "user", content: "Review this diff." },
  ],
});

File: test/fixtures/crossmodule-repo/agent.py
import openai

from prompts import SYSTEM_PROMPT

client = openai.OpenAI()

resp = client.chat.completions.create(
    model="gpt-4o",
    messages=[
        {"role": "system", "content": SYSTEM_PROMPT},
        {"role": "user", "content": "Please help with my billing issue."},
    ],
)
